contracts/vaults/common/VaultTypes.sol

```diff
@@ -96,4 +105,12 @@ struct ThreeTokenPoolContext {
     uint8 tertiaryDecimals;
```

Comment:
@weitianjie2000 ThreeTokenPoolContext can be removed from this file as well.

Reply:
removed